lib/Models/Catalog/Gltf/AssImpCatalogItem.ts

```diff
@@ -197,22 +210,22 @@ export default class AssImpCatalogItem
         gltfJson.images?.forEach((image) => {
           if (!image.uri) return;
 
+          // Replace back slashes with forward slash
+          let newUrl = image.uri.replace(/\\/g, "/");
+          // Remove start "./" or "//" from uri
+          if (newUrl.startsWith("//") || newUrl.startsWith("./")) {
+            newUrl = newUrl.slice(2);
+          }
+
```

Comment:
We were already doing this rewrite, but after the supported extensions check. This resulted in incorrect warning about unsupported extensions because `URI('\\texture.png')` treats `texture.png` as domain name instead of path.